hack/components.sh

```diff
@@ -116,3 +120,58 @@ MACVTAP_IMAGE_TAGGED=${MACVTAP_IMAGE}:${MACVTAP_TAG}
 sed -i "s#\"${MACVTAP_IMAGE}:.*\"#\"${MACVTAP_IMAGE_TAGGED}\"#" pkg/components/components.go
 # TODO: uncomment the following line *once* there is macvtap upgrade is supported
 #sed -i "s#\"${MACVTAP_IMAGE}:.*\"#\"${MACVTAP_IMAGE_TAGGED}\"#" test/releases/${CNAO_VERSION}.go
+
+echo 'linux-bridge'
+LINUX_BRIDGE_URL=$(cat components.yaml | shyaml get-value components.linux-bridge.url)
+LINUX_BRIDGE_COMMIT=$(cat components.yaml | shyaml get-value components.linux-bridge.commit)
+LINUX_BRIDGE_REPO=$(echo ${LINUX_BRIDGE_URL} | sed 's#https://\(.*\)#\1#')
+LINUX_BRIDGE_PATH=${GOPATH}/src/${LINUX_BRIDGE_REPO}
+
+echo 'Fetch linux-bridge sources'
+(
+    mkdir -p ${LINUX_BRIDGE_PATH}
+    git clone ${LINUX_BRIDGE_URL} ${LINUX_BRIDGE_PATH} || true
+    cd ${LINUX_BRIDGE_PATH}
+    git fetch origin
+    git reset --hard
+    git checkout ${LINUX_BRIDGE_COMMIT}
+)
```

Comment:
We should eventually move this into a function - I also didn't do it, so I don't think it's fair to make you jump through that hoop.

Reply:
Good call

https://github.com/kubevirt/cluster-network-addons-operator/pull/367